solutions/saas-microservices/apps/dashboard/src/middleware.ts

```diff
@@ -0,0 +1,24 @@
+import { NextRequest, NextResponse } from "next/server";
+
+export function middleware(request: NextRequest) {
+  const currentUrl = request.nextUrl;
+  const authedUser = request.cookies.get("saas_microservices_authed_user");
```

Comment:
super minor nit but this cookie could be exposed in a shared package so all of the apps can use the same value, but feel free to ignore